mac/macdeploy.sh

```diff
@@ -19,23 +20,50 @@ cd bin
 # copy libQupZilla into bundle
 cp $LIBRARY_NAME QupZilla.app/Contents/MacOS/
 
-# copy all plugins into bundle
-test -d QupZilla.app/Contents/Resources/plugins || mkdir QupZilla.app/Contents/Resources/plugins
-cp plugins/*.dylib QupZilla.app/Contents/Resources/plugins/
+# copy all QupZilla plugins into bundle
+test -d $PLUGINS || mkdir $PLUGINS
+cp plugins/*.dylib $PLUGINS/
 
 # fix libQupZilla
 install_name_tool -change $LIBRARY_NAME @executable_path/$LIBRARY_NAME QupZilla.app/Contents/MacOS/QupZilla
 
 # fix plugins
-for plugin in QupZilla.app/Contents/Resources/plugins/*.dylib
+for plugin in $PLUGINS/*.dylib
```

Comment:
> This should not be done for Qt plugins.

Rereading this a few times to digest a possible singular meaning... I didn't set that for the Qt plugin *(the libqsvgicon.dylib)* e.g. it's a flat out copy *(does work if I do set it though but decided it probably wouldn't be the right thing to do)*... so there is no issue here.  Read further down in the shell script.